src/offset/local/tz_info/timezone.rs

```diff
@@ -572,8 +584,11 @@ impl LocalTimeType {
     }
 
     /// Returns offset from UTC in seconds
-    pub(crate) const fn offset(&self) -> i32 {
-        self.ut_offset
+    pub(crate) const fn offset(&self) -> Result<FixedOffset, Error> {
+        match FixedOffset::east_opt(self.ut_offset) {
```

Comment:
This is kind of fake, right? Don't we already know, by here, that `ut_offset` is in bounds?

Reply:
If it was my personal code I would just trust the timezone data to be valid an `unwrap` all over the place :smile:. It is mostly there because `unwrap` and `expect` are not available in `const`.

This preserves the check that was in https://github.com/chronotope/chrono/blob/main/src/offset/local/unix.rs#L156

Reply:
I'll try to find a better way to structure this.

Reply:
If we execute on the MSRV bump discussed in #995 (which we should be doing anyway -- hint hint), `unwrap()` might become available in `const` contexts. However, the issue with `unwrap()` in `const` is that, while I'm fine with `unwrap()` being called in an actual `const` context (that is, at compile time), `const` functions can actually still also be called at run time, so we still don't want to panic in that case.

Reply:
I have made `LocalTimeType` store a `FixedOffset`, so validation is now moved to creation time instead of use.